### ISSUE TITLE
Zoom in/out the server map in the transaction detail view to fit the available width

### ISSUE THREAD

**Issue by yjqg6666** [naver/pinpoint#6793]
**Which module is your feature request related to?**
UI

**Is your feature request related to a problem?**
The server map ie. [this one](http://125.209.240.10:10123/transactionView/apigw01/apigw01%5E1588905885082%5E2161981/1589452764325/6215127377226995229), the UI looks like this:
![image](https://user-images.githubusercontent.com/1879641/81925847-a2e9d580-9613-11ea-81b8-4d28817142e1.png)
It could be zoomed out by default for the full view.
![image](https://user-images.githubusercontent.com/1879641/81925961-d3ca0a80-9613-11ea-9093-ee72b85b2771.png)

**Describe the solution you'd like**
There should be some APIs for cytoscape library to adjust the canvas.

**Comment by yjqg6666** [comment]
The same issue to Server Map tab.
![image](https://user-images.githubusercontent.com/1879641/81926549-b2b5e980-9614-11ea-8743-32a613d409ee.png)

**Comment by yjqg6666** [comment]
https://js.cytoscape.org/#cy.fit 
cy.fit() may need be called after rendering done.

**Comment by yjqg6666** [comment]
From the source code [\#1](https://github.com/naver/pinpoint/blob/master/web/src/main/angular/src/app/core/components/server-map/class/server-map-diagram-with-cytoscapejs.class.ts#L102-L105) and [#2](https://github.com/naver/pinpoint/blob/master/web/src/main/angular/src/app/core/components/server-map/class/server-map-diagram-with-cytoscapejs.class.ts#L265-L267) i found that it would be zoomed (this.cy.zoom(1) is called) when the layout is ready. 

When view [this page](http://125.209.240.10:10123/transactionView/apigw01/apigw01%5E1588905885082%5E2161981/1589452764325/6215127377226995229), i got many errors found in the browser console:

![image](https://user-images.githubusercontent.com/1879641/82004791-57c4d680-9696-11ea-92f2-b3e4c5fb7611.png)

![image](https://user-images.githubusercontent.com/1879641/82004702-251ade00-9696-11ea-9fbf-7f0f417c9c91.png)

![image](https://user-images.githubusercontent.com/1879641/82004298-17188d80-9695-11ea-8ed9-9b20a6ef9298.png)

**Comment by binDongKim** [comment]
@yjqg6666 I also considered using `fit()` method when I first implemented this but the reason I chose not to use is because of the variety of the nodes size depending on the application. If an app has a couple of nodes, the `fit()` method may work okay but if it has quite a complicated structure server map, the method tries to put all nodes in the view making things invisibly small. So, I rather set the size of the node and centralize the *main* node. But I'll consider it again later.
Thanks.

Plus, I didn't get to see the errors.
